tests/v2/e2e/config/config.go

```diff
@@ -68,15 +69,15 @@ type Operation struct {
 
 // Execution represents the execution details for a given operation.
 type Execution struct {
-	*BaseConfig         `                    yaml:",inline,omitempty"               json:",inline,omitempty"`
-	TimeConfig          `                    yaml:",inline,omitempty"               json:",inline,omitempty"`
-	Name                string              `yaml:"name"                            json:"name,omitempty"`
-	Type                OperationType       `yaml:"type"                            json:"type,omitempty"`
-	Mode                OperationMode       `yaml:"mode"                            json:"mode,omitempty"`
-	Search              *SearchQuery        `yaml:"search,omitempty"                json:"search,omitempty"`
-	Kubernetes          *KubernetesConfig   `yaml:"kubernetes,omitempty"            json:"kubernetes,omitempty"`
-	Modification        *ModificationConfig `yaml:"modification,omitempty"          json:"modification,omitempty"`
-	ExpectedStatusCodes StatusCodes         `yaml:"expected_status_codes,omitempty" json:"expected_status_codes,omitempty"`
+	*BaseConfig  `                    yaml:",inline,omitempty"      json:",inline,omitempty"`
+	TimeConfig   `                    yaml:",inline,omitempty"      json:",inline,omitempty"`
+	Name         string              `yaml:"name"                   json:"name,omitempty"`
+	Type         OperationType       `yaml:"type"                   json:"type,omitempty"`
+	Mode         OperationMode       `yaml:"mode"                   json:"mode,omitempty"`
+	Search       *SearchQuery        `yaml:"search,omitempty"       json:"search,omitempty"`
+	Kubernetes   *KubernetesConfig   `yaml:"kubernetes,omitempty"   json:"kubernetes,omitempty"`
+	Modification *ModificationConfig `yaml:"modification,omitempty" json:"modification,omitempty"`
+	Expect       *Expect             `yaml:"expect,omitempty"       json:"expect,omitempty"`
```

Comment:
How about allowing multiple pattern
```suggestion
	Expects       []Expect             `yaml:"expects,omitempty"       json:"expects,omitempty"`
```

Reply:
Currently using `results` for multiple pattern. I failed to push [the change](https://github.com/vdaas/vald/pull/2971/commits/bca5715f1cd1c9a7ab2fd0fbfc58f26d2b139b7f) 🙏

Reply:
Each Response has its own StatusCode and I think they are paired together, so is it necessary to expect multiple Responses with a single StatusCode?

Reply:
Thank you, let's pair the statuscode and result.
Do you mean something like
```
expects:
  - response: {}
  - statuscode: 404
  - statuscode: 200
    response:
      stored: 3000
```

Reply:
@kpango Handled in [this commit](https://github.com/vdaas/vald/pull/2971/commits/e85a9bff5585b98636c2860ea69cbe041feaf4bd).